.static_files

```diff
@@ -19,6 +19,7 @@ scripts/README.md
 .github/workflows/check_pr_target.yaml
 .github/workflows/static_code_analysis.yaml
 .github/workflows/check_mandatory_and_static_files.yaml
+.github/workflows/unit_and_int_tests.yaml
```

Comment:
You are right `check_config_schema_and_example.yaml` can be included here. I just did it.

However, for the rest it's not possible because:

- `check_db_migration_scripts.yaml`  is only required for SQL-based services
- `check_openapi_spec.yaml` is only required for services with an openapi-spec